### ISSUE TITLE
Bootstrap script fails: "Could not find or load main class com.kodcu.boot.AppStarter"

### ISSUE THREAD

**Issue by cengiz-io** [asciidocfx/AsciidocFX#12]
Hi there,

Just downloaded [v1.0.10](https://github.com/rahmanusta/AsciidocFX/releases/tag/v1.0.10) zip.

`bin/asciidocfx.sh` fails with main class not found error:

```
➜ asciidocfx: bin/asciidocfx.sh 
Error: Could not find or load main class com.kodcu.boot.AppStarter
```

Here's the command line that `exec` statement trying to execute

```
/usr/lib/jvm/default/bin/java -classpath /tmp/asciidocfx/conf:/tmp/asciidocfx/lib/* -Dapp.name=asciidocfx -Dapp.pid=32707 -Dapp.repo=/tmp/asciidocfx/lib -Dapp.home=/tmp/asciidocfx -Dbasedir=/tmp/asciidocfx com.kodcu.boot.AppStarter
```

I suspect an incompatibility with OpenJDK / OpenJFX but the error message is very unlikely.

**FYI: I'm using OpenJDK 8b20**

Keep up the good work

Saygılar

**Comment by wget** [comment]
@cengizIO While it works with asciidocfx 1.5.5, I'm experiencing this bug again on Arch Linux when trying to run asciidocfx 1.5.6.
@rahmanusta Any idea would be welcome.
I'm in the process of upgrading OpenJFX on ArchLinux as well. Were is the PKGBUILD I made: https://github.com/wget/archlinux-package-java-openjfx

**Comment by wget** [comment]
Fixed by backporting changes committed after 1.5.6 has been tagged.
